.yamllint.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+extends: default
+
+rules:
```

Comment:
Based on https://developers.google.com/devsite/reference/metadata/yaml#nested_yaml_elements
I believe we should add:
```
  indentation:
    spaces: 2
    indent-sequences: false
```

Reply:
SGTM